src/components/ui/ListEntry.js

```diff
@@ -8,7 +8,7 @@ const Icons = styled.div`
   align-items: center;
   max-height: 36px;
   & > * {
-    padding: 8px;
+    margin: 8px;
```

Comment:
Should this be horizontal margin only?

Reply:
Also it looks like a lot of margin in the screenshot, double-check that with Figma.

Reply:
Good point. Distance looks too big